package.json

```diff
@@ -50,21 +50,25 @@
     "sanitize.css": "^5.0.0",
     "seedrandom": "^2.4.4",
     "sharetribe-flex-sdk": "^1.3.0",
-    "sharetribe-scripts": "2.1.8",
+    "sharetribe-scripts": "3.0.0-rc",
     "smoothscroll-polyfill": "^0.4.0",
     "source-map-support": "^0.5.9",
     "url": "^0.11.0"
   },
   "devDependencies": {
+    "babel-jest": "24.7.1",
```

Comment:
There was a strange transient dependency problem without this. So, I had to include it as explicit dev-dependency:
https://github.com/timarney/react-app-rewired/issues/388